bin/varnishd/cache/cache_backend.c

```diff
@@ -125,9 +126,12 @@ vbe_dir_getfd(struct worker *wrk, struct backend *bp, struct busyobj *bo,
 	CHECK_OBJ_NOTNULL(bp, BACKEND_MAGIC);
 	AN(bp->vsc);
 
+	bo->fail_reason = BE_FAIL_UNKNOWN;
```

Comment:
is this really the best place for the initialization? I wonder if it should rather be moved to when we create the busyobj

Reply:
Makes sense. Thanks for bringing it up. I have moved it.